source/_docs/traffic-limits.md

```diff
@@ -10,14 +10,17 @@ Unlike traditional providers where enough requests will crash the server, Panthe
 However, this does mean that websites can run happily even though they are over their plan's traffic limit. If this is happening on a sustained basis, you may receive notice from customer support that your site's plan isn't adequate to its needs, and an upgrade may be in order.
 
 ## How We Measure Site Traffic
+{% include("content/traffic-dl.html")%}
 
-Pantheon defines plan levels based on pageviews to help site owners pick the right plan based on expected or historical traffic. To verify that sites receive traffic within their plan limit, we count requests served by the platform from the CMS. Cached responses count as pageviews.
+For more details, see [Metrics in the Site Dashboard](/docs/metrics/).
+
+Pantheon defines plan levels based on **pages served** to help site owners pick the right plan based on expected or historical traffic. To verify that sites receive traffic within their plan limit, we count requests served by the platform from the CMS. Cached responses count as pages served.
```

Comment:
The last two sentences in this paragraph seem co contradict each other, since cached responses don't hit the CMS.

Are we trying to say that "we count requests served by the platform, either cached or directly from the CMS"?

Reply:
Cached responses are originally generated by the CMS. Does this clarify?
"we count requests served by the platform that are originally generated by the CMS."